test/acceptance/specs/EmailConfirmedSpec.scala

```diff
@@ -66,7 +68,8 @@ class EmailConfirmedSpec extends BaseSpec
       save4LaterWithNoData("123456789")
       navigateTo(StartPage)
       verifyCurrentPage(StartPage)
-      authenticateGGUserAndReturnEoriEnrolment("123456789")
+      stubSubscriptionDisplayOkResponse()
```

Comment:
can we verify that the number of calls made to subscription display(customs-hods-proxy) here, please

Reply:
will be addressed as part of testing